matchmaker/views/external_api_tests.py

```diff
@@ -12,6 +12,7 @@
 
 @mock.patch('matchmaker.views.external_api.MME_NODES', TEST_MME_NODES)
 class ExternalAPITest(TestCase):
+    databases = '__all__'
     fixtures = ['users', '1kg_project', 'reference_data']
     multi_db = True
```

Comment:
can you remove multi_db here

Reply:
Sorry, I thought `multi_db = True` was still required.